--- openal/linux/configure.in.orig	Mon Jan 13 23:28:25 2003
+++ openal/linux/configure.in	Tue Feb 18 21:24:56 2003
@@ -85,23 +85,7 @@
 enable_paranoid_locks=no
 enable_empty_locks=no
 
-CFLAGS="-O6 -fexpensive-optimizations -funroll-all-loops -funroll-loops -fomit-frame-pointer -finline-functions -ffast-math $CFLAGS"
-
-dnl add -march
-case "$target" in 
-    *i386*)
-    CFLAGS="$CFLAGS -march=i386" ;;
-    *i486*)
-    CFLAGS="$CFLAGS -march=i486" ;;
-    *i586*)
-    CFLAGS="$CFLAGS -march=i586" ;;
-    *i686*)
-    CFLAGS="$CFLAGS -march=i686" ;;
-    mips*)
-    CFLAGS="$CFLAGS -march=mips" ;;
-    *)
-    echo "Unknown target $target" ;;
-esac
+CFLAGS="-fexpensive-optimizations -funroll-all-loops -funroll-loops -fomit-frame-pointer -finline-functions -ffast-math $CFLAGS"
 
 fi
 
@@ -270,7 +254,7 @@
 [  --enable-dmalloc          enable DMALLOC support       [default=no]],
               , enable_dmalloc=no)
 if test x$enable_dmalloc = xyes; then
-    AC_CHECK_HEADER(dmalloc.h, 
+    AC_CHECK_HEADER(dmalloc.h,[
     	dnl header found
 	AC_DEFINE(DMALLOC, 1, [undocumented])
 	AC_DEFINE(DMALLOC_FUNC_CHECK, 1, [undocumented])
@@ -280,14 +264,14 @@
     	REDUNDANT_DECLS_WARN=no,
 	dnl header not found, check local/include
 	dnl invalid cache
-	[AC_CHECK_HEADER(/usr/local/include/dmalloc.h,
+	AC_CHECK_HEADER(/usr/local/include/dmalloc.h,[
 		dnl header found in local/include
 		AC_DEFINE(DMALLOC, 1, [undocumented])
 		AC_DEFINE(DMALLOC_FUNC_CHECK, 1, [undocumented])
 		INCLUDES="$CFLAGS -include \"/usr/local/include/dmalloc.h\""
 		LIBS="$LIBS -ldmallocth"
 		SHOULD_USE_WERROR=no
-		REDUNDANT_DECLS_WARN=no)])
+		REDUNDANT_DECLS_WARN=no])])
 fi
 
 dnl efence stuff
@@ -685,7 +669,7 @@
 if test x$enable_vorbis = xyes; then
 
 	AC_CHECK_LIB(vorbisfile, ov_info,
-		AC_DEFINE( VORBIS_SUPPORT, 1, [undocumented] )
+		AC_DEFINE([VORBIS_SUPPORT], 1, [undocumented] )
 		LIBS="$LIBS -lvorbis -lvorbisfile",, -lvorbis )
 
 fi #   enable_vorbis = no
