--- openal-soft-1.24.1/fmt-11.0.2/CMakeLists.txt.orig	2024-11-28 04:21:16.000000000 +0100
+++ openal-soft-1.24.1/fmt-11.0.2/CMakeLists.txt	2024-12-01 09:08:16.507841204 +0100
@@ -66,7 +66,7 @@ add_headers(FMT_HEADERS args.h base.h ch
                         xchar.h)
 set(FMT_SOURCES src/format.cc src/os.cc)
 
-add_library(alsoft.fmt ${FMT_SOURCES} ${FMT_HEADERS} README.md ChangeLog.md)
+add_library(alsoft.fmt STATIC ${FMT_SOURCES} ${FMT_HEADERS} README.md ChangeLog.md)
 add_library(alsoft::fmt ALIAS alsoft.fmt)
 
 if (cxx_std_11 IN_LIST CMAKE_CXX_COMPILE_FEATURES)
