diff -urN openal_/linux/Makefile.in openal/linux/Makefile.in
--- openal_/linux/Makefile.in	2005-07-14 13:42:27.000000000 +0200
+++ openal/linux/Makefile.in	2005-07-14 13:42:47.000000000 +0200
@@ -1,9 +1,9 @@
 INSTALL=@INSTALL@
 LN_S=@LN_S@
-DESTDIR=@prefix@/
-DESTINC=$(DESTDIR)include
-DESTLIB=$(DESTDIR)lib
-DESTBIN=$(DESTDIR)bin
+DESTDIR=
+DESTINC=$(DESTDIR)@prefix@/include
+DESTLIB=$(DESTDIR)@prefix@/lib
+DESTBIN=$(DESTDIR)@prefix@/bin
 DESTPKG=$(DESTLIB)/pkgconfig/
 PACKAGE_VERSION_MAJOR=@PACKAGE_VERSION_MAJOR@
 PACKAGE_VERSION=@PACKAGE_VERSION@
