--- openal/linux/Makefile.in.orig	Fri Sep  6 00:54:36 2002
+++ openal/linux/Makefile.in	Tue Feb 18 21:22:22 2003
@@ -1,8 +1,10 @@
 INSTALL=@INSTALL@
 LN_S=@LN_S@
-DESTDIR=@prefix@/
-DESTINC=$(DESTDIR)include
-DESTLIB=$(DESTDIR)lib
+includedir=@includedir@
+libdir=@libdir@
+DESTDIR=
+DESTINC=$(DESTDIR)$(includedir)
+DESTLIB=$(DESTDIR)$(libdir)
 MAKE=@MAKE@
 MAJOR_VERSION=@MAJOR_VERSION@
 MINOR_VERSION=@MINOR_VERSION@
@@ -52,8 +54,8 @@
 install_minor_libs:
 	$(INSTALL) -d -m 755 $(DESTLIB)
 	$(INSTALL) -m 755 src/libopenal.$(SO_EXT).$(VERSION) $(DESTLIB)
-	$(LN_S) -f $(DESTLIB)/libopenal.$(SO_EXT).$(VERSION) $(DESTLIB)/libopenal.$(SO_EXT).$(MAJOR_VERSION)
-	$(LN_S) -f $(DESTLIB)/libopenal.$(SO_EXT).$(VERSION) $(DESTLIB)/libopenal.so
+	$(LN_S) -f libopenal.$(SO_EXT).$(VERSION) $(DESTLIB)/libopenal.$(SO_EXT).$(MAJOR_VERSION)
+	$(LN_S) -f libopenal.$(SO_EXT).$(VERSION) $(DESTLIB)/libopenal.so
 
 install_major_lib_only:
 	$(INSTALL) -d -m 755 $(DESTLIB)
