--- openal-soft-1.14/CMakeLists.txt.orig	2012-03-28 17:55:17.000000000 +0200
+++ openal-soft-1.14/CMakeLists.txt	2012-04-04 17:43:01.116501828 +0200
@@ -663,7 +663,7 @@
 
 IF(EXAMPLES)
     # Might be able to use earlier versions, but these definitely work
-    PKG_CHECK_MODULES(FFMPEG libavcodec>=52.123.0 libavformat>=52.111.0)
+    PKG_CHECK_MODULES(FFMPEG libavcodec>=52.123.0 libavformat>=52.111.0 libavutil)
 ENDIF()
 
 IF(LIBTYPE STREQUAL "STATIC")
